com.unity.render-pipelines.core/Tests/Editor/Volumes/VolumeComponentTests.cs

```diff
@@ -159,5 +160,23 @@ public void TestHandleParameterDecorators()
         }
 
         #endregion
+
+        [Test]
+        public void TestVolumeComponentProvider()
```

Comment:
What is the aim of this test? In my understanding it checks if `FilterVolumeComponentTypes` returns an empty dictionary when it receives some VolumeComponets . Would it not also be useful to test if we get the correct result, e.g. get a component when it is supported by the render pipeline and a result is discarded when it is not?

Reply:
Changed the test, checking that the `SupportedOn `does not avoid the skipping if there is `HideInInspector `defined.